<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>259</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>259</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>259</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="okButtonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>81</width>
     <height>31</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="appMetaDataLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>450</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>151</height>
    </size>
   </property>
   <property name="text">
    <string>appMetaData</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="copyrightClogo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>(C) Chris Luu 2023</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
